packages/client/wallets/aa/src/utils/user.ts

```diff
@@ -39,6 +28,19 @@ export function parseUserIdentifier(reqParams: UserIdentifierParams, acceptPhone
     throw new Error(`Missing required parameter.`);
 }
 
+export function getIdString(identifier: UserIdentifier): string {
+    switch (identifier.type) {
+        case "whiteLabel":
+            return identifier.userId;
+        case "email":
+            return identifier.email;
+        case "phoneNumber":
+            return identifier.phoneNumber;
+        default:
+            throw new Error("Invalid UserIdentifier type.");
```

Comment:
TBH I'd rather omit this and let the compiler fail if we add a case and forget to handle it